scripts/altair_chart_definitions.py

```diff
@@ -90,4 +90,31 @@
 ).configure_title(
             anchor='middle'
         )
-chart
\ No newline at end of file
+chart
+
+
+
+
+def get_chart(percentile_data, top_n_data, col_name):
```

Comment:
You can ignore this file, i use it as a dumping ground for the Altair chart definitions I used to create the vega lite specs